### PR TITLE
Reddit word Cloud

### PR DESCRIPTION
**Pull request by rayye1**
- Got a word of comments that the user uses

### CODE REVIEW

File: frontend/src/components/Reddit/RedditComments.js [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
...ar","did","do","does","either","else","ever","every","for","from","get","got","had","has","have","he","her","hers","him","his","how","however","i","if","in","into","is","it","its","just","least","let","like","likely","may","me","might","most","must","my","neither","no","nor","not","of","off","often","on","only","or","other","our","own","rather","said","say","says","she","should","since","so","some","than","that","the","their","them","then","there","these","they","this","tis","to","too","twas","us","wants","was","we","were","what","when","where","which","while","who","whom","why","will","with","would","yet","you","your","ain't","aren't","can't","could've","couldn't","didn't","doesn't","don't","hasn't","he'd","he'll","he's","how'd","how'll","how's","i'd","i'll","i'm","i've","isn't","it's","might've","mightn't","must've","mustn't","shan't","she'd","she'll","she's","should've","shouldn't","that'll","that's","there's","they'd","they'll","they're","they've","wasn't","we'd","we'll","we're","weren't","what'd","what's","when'd","when'll","when's","where'd","where'll","where's","who'd","who'll","who's","why'd","why'll","why's","won't","would've","wouldn't","you'd","you'll","you're","you've"]
```
Comment (tappenze) [inline frontend/src/components/Reddit/RedditComments.js]: what is this? Can we put this somewhere else?

File: frontend/src/components/Reddit/RedditPage.js [hunk 2]
```diff
@@ -1,16 +1,63 @@
...
...ar","did","do","does","either","else","ever","every","for","from","get","got","had","has","have","he","her","hers","him","his","how","however","i","if","in","into","is","it","its","just","least","let","like","likely","may","me","might","most","must","my","neither","no","nor","not","of","off","often","on","only","or","other","our","own","rather","said","say","says","she","should","since","so","some","than","that","the","their","them","then","there","these","they","this","tis","to","too","twas","us","wants","was","we","were","what","when","where","which","while","who","whom","why","will","with","would","yet","you","your","ain't","aren't","can't","could've","couldn't","didn't","doesn't","don't","hasn't","he'd","he'll","he's","how'd","how'll","how's","i'd","i'll","i'm","i've","isn't","it's","might've","mightn't","must've","mustn't","shan't","she'd","she'll","she's","should've","shouldn't","that'll","that's","there's","they'd","they'll","they're","they've","wasn't","we'd","we'll","we're","weren't","what'd","what's","when'd","when'll","when's","where'd","where'll","where's","who'd","who'll","who's","why'd","why'll","why's","won't","would've","wouldn't","you'd","you'll","you're","you've"]
```
Comment (tappenze) [inline frontend/src/components/Reddit/RedditPage.js]: same here. In fact, I'm noticing a lot of the same functions, can we put them in their own file and import them?

**Review by tappenze — CHANGES_REQUESTED** [review]

File: frontend/src/components/Reddit/RedditComments.js (hunk 1 above)
Comment (rayye1) [reply frontend/src/components/Reddit/RedditComments.js]: Its a list of common words to ignore. I just found it easier to keep it in thsi file

File: frontend/src/components/Reddit/RedditPage.js (hunk 2 above)
Comment (rayye1) [reply frontend/src/components/Reddit/RedditPage.js]: which functions are you referring to?

Comment (tappenze) [reply frontend/src/components/Reddit/RedditPage.js]: getuncommon and getwordlist

**Review by tappenze — APPROVED** [review]
lgtm

**Review by chriskoehne — APPROVED** [review]
resolved merge conflicts

**Pull request merged**